dashboard-for-desktop/articles/dashboard-designer/appearance-customization/coloring/coloring-concepts.md

```diff
@@ -3,61 +3,96 @@ title: Coloring Concepts
-Dashboard items allow you to manage the coloring of individual dimensions or all dashboard item measures using predefined coloring modes.
-
-| Coloring Mode | Description |
-|---|---|
-| Default | Dimension values/measures are colored by default. To learn how specific dashboard items color their elements by default, see the **Coloring** topic for the corresponding dashboard item. |
-| Hue | Dimension values/measures are colored by hue. If coloring by hue is enabled, a data item indicates this using the ![ColoringIndicator](../../../../images/img25453.png) indicator. |
-| None | Dimension values/measures are colored with the same color. |
-
-### Coloring Dimension Values
-
-To specify the coloring mode for the required dimension, click the dimension's [menu button](../../ui-elements/data-items-pane.md) and use the **Color by** submenu. For instance, the image below shows the Chart dashboard item whose 'Country' dimension is colored by hue.
-
-![Coloring_DimensionColorByItem](../../../../images/img25374.png)
-
-### Coloring Measures
-
-To specify the coloring mode for dashboard item measures, click the [menu button](../../ui-elements/data-items-pane.md) of any measure and use the **Color by** submenu. For instance, the image below shows the Pie dashboard item whose measures are colored by hue.
-
-![Coloring_MeasuresColorByItem](../../../../images/img25376.png)
-
-If you enabled coloring by hue for several dimensions/measures, all combinations of dimension values/measures will be automatically colored using different colors from the default palette. To learn how to customize these colors, see [Customizing a Color Scheme](customizing-a-color-scheme.md).
\ No newline at end of file
+# Coloring 
+
+The Dashboard Designer associates dimension values/measures and specified colors to paint dashboard item elements. This topic describes how to configure color settings in the WinForms Designer.
+
+## Coloring Basics
+
+The following concepts are common to both Desktop and Web Dashboard controls:
+
+* Color Modes
+* Color Schemes
+* Supported Dashboard Items
+
+Refer to the following topic for more information about common concepts: [Coloring Basics](../coloring.md).
+
+## Color Mode: None 
+
+You can disable default color variation for dashboard item elements.  
+
+If you add a [TreeMap](../../designing-dashboard-items/treemap.md) to a dashboard, individual elements (titles) use different colors: 
+
+![win-coloring-treemap-default](../../../../images/win-coloring-treemap-default.png)
+
+_State_ values (Arguments) use different colors. To disable color variation, go to the Argument settings and select **Color by | None**.
+
+![win-coloring-treemap-none](../../../../images/win-coloring-treemap-none.png)
+
+Add a [Chart](../../designing-dashboard-items/chart.md) with the same Argument and Value as in the TreeMap: 
+
+![win-coloring-treemap-chart-none](../../../../images/win-coloring-treemap-chart-none.png)
+
+**Default** means **None** for chart arguments.  
```

Comment:
To make this sentence *flow* with the previous sentence above, should we say:

`Note that **Default** means **None** for chart arguments. `